apps/omg_watcher_rpc/coveralls.json

```diff
@@ -1,5 +1,6 @@
 {
   "skip_files": [
-    "test/support"
+    "test/support",
```

Comment:
I guess we maybe want to wait until https://github.com/omisego/elixir-omg/pull/889 is merged so these can be moved/removed.

Reply:
Yep!